src/uct/ib/base/ib_iface.h

```diff
@@ -216,6 +216,7 @@ typedef struct uct_ib_qp_attr {
 
 typedef ucs_status_t (*uct_ib_iface_create_cq_func_t)(uct_ib_iface_t *iface,
                                                       uct_ib_dir_t dir,
+                                                      const uct_ib_iface_config_t *config,
```

Comment:
Maybe add `cqe_zipping_enabled` to `uct_ib_iface.config` instead of passing the `config` in `create_cq` (and modifying many places)?
@yosefe  WDYT?